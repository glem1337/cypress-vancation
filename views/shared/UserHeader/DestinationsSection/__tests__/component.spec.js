import React from 'react';
import { shallow } from 'enzyme';

import DestinationSection from '../component';

describe('DestinationSection component tests', () => {
  const component = shallow(<DestinationSection />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('matches snapshot when is not visible', () => {
    component.setProps({ isVisible: false });

    expect(component).toMatchSnapshot();
  });
});
