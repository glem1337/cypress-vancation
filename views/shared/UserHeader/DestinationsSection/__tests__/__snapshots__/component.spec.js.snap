// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DestinationSection component tests matches snapshot 1`] = `
<div
  className="main-account-header__item ml-auto"
>
  <span
    className="mr-4 main-account-header__item-txt"
  >
    <Memo(FormattedMessage)
      id="headerHeader.destinations"
    />
  </span>
  <i
    className="icon icon-down in-black font-16"
  />
</div>
`;

exports[`DestinationSection component tests matches snapshot when is not visible 1`] = `""`;
